modules/tensor_mechanics/tests/elastic_patch/tests

```diff
@@ -1,7 +1,21 @@
 [Tests]
-  [./elastic_patch]
+  [./test_elastic_patch]
```

Comment:
This is already merged, but I'm not a fan of changes like this. The user _knows_ this is a "test". There is no reason to name the test "test..".